pkg/frontend/adminactions/azureactions.go

```diff
@@ -69,3 +71,13 @@ func (a *azureActions) VMRedeployAndWait(ctx context.Context, vmName string) err
 	clusterRGName := stringutils.LastTokenByte(a.oc.Properties.ClusterProfile.ResourceGroupID, '/')
 	return a.virtualMachines.RedeployAndWait(ctx, clusterRGName, vmName)
 }
+
+func (a *azureActions) VMRestartAndWait(ctx context.Context, vmName string) error {
+	clusterRGName := stringutils.LastTokenByte(a.oc.Properties.ClusterProfile.ResourceGroupID, '/')
+	return a.virtualMachines.StartAndWait(ctx, clusterRGName, vmName)
```

Comment:
Why do we need both `VMRestartAndWait` and `VMStartAndWait` when both methods end up calling `StartAndWait`?